Gemfile

```diff
@@ -26,6 +26,10 @@ group :test do
 
   # docker-api 1.32.0 requires ruby 2.0.0
   gem 'docker-api', '1.31.0'
+
+  # The public_suffix gem used in the dependencies for the packaging gem no longer supports Ruby 2.5
+  # and bundler is not smart enough to realize that an older version would work in the mix.
+  gem 'public_suffix', '= 4.0.7'
```

Comment:
This might need to go outside the `:test` block since I think it's `packaging` that pulls in `public_suffix`